packages/circuit-ui/components/SelectorGroup/SelectorGroup.tsx

```diff
@@ -52,12 +57,100 @@ export interface SelectorGroupProps {
    * Whether the user can select multiple options.
    */
   multiple?: boolean;
+  /**
+   * Size of the Selectors within the group. Default: 'mega'.
+   */
+  size?: SelectorSize;
+  /**
+   * Spacing between selectors in a group. Default: 'kilo'.
+   */
+  gapSize?: GapSize;
+  /**
+   * Whether to display Selectors inline on mobile. Default: false.
+   */
+  inlineMobile?: boolean;
+  /**
+   * Whether the group should tale the whole width available when displayed in a row. Default: false.
+   */
+  stretch?: boolean;
```

Comment:
Not sure we should expose these two 🤔 it might be better to let consumers add custom styles for edge cases. What do you think?

Reply:
At first, I thought exactly that as well, to leave it up to the user. But the issue is that `Selector` needs to be modified as well in order to be able to stretch (see `StyledSelector`), so it does not really make sense to keep in mind this stretching usecase by adjusting internal elements but to not expose the top-level property.